.github/workflows/ci-release.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     name: Release JDK ${{ matrix.java }}
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
for snapshots I see the point, would there be any negative side effects we release one but not the other potentially?

Reply:
no, the set of produced modules is completely independent and because of the occasional flaky tests it affects the other pipeline